src/components/PrivateRoute.tsx

```diff
@@ -1,28 +1,17 @@
 import { FunctionComponent } from "react";
 import { useSelector } from "react-redux";
 import { selectSession } from "modules/session";
-import { Redirect, Route, RouteProps } from "react-router-dom";
+import { Navigate } from "react-router-dom";
 
-const PrivateRoute: FunctionComponent<RouteProps> = ({ children, ...rest }) => {
+type PrivateRouteProps = {
+  children: JSX.Element;
+};
+
+const PrivateRoute: FunctionComponent<PrivateRouteProps> = ({ children }) => {
```

Comment:
Because we will now always be wrapping `<PrivateRoute>` components in a `<Route>` component, there is no need for rendering a `<Route>` component